FLT/HaarMeasure/ConcreteCharCalculations.lean

```diff
@@ -0,0 +1,79 @@
+import Mathlib.Analysis.Complex.ReImTopology
+import Mathlib.MeasureTheory.Measure.Lebesgue.EqHaar
+import Mathlib.RingTheory.Complex
+import Mathlib.RingTheory.Localization.NumDen
+import Mathlib.Analysis.Normed.Group.Ultra
+import FLT.HaarMeasure.DistribHaarChar
+import FLT.Mathlib.Analysis.Complex.Basic
+import FLT.Mathlib.Data.Complex.Basic
+import FLT.Mathlib.LinearAlgebra.Determinant
+import FLT.Mathlib.MeasureTheory.Group.Action
+import FLT.Mathlib.NumberTheory.Padics.PadicIntegers
+import FLT.Mathlib.NumberTheory.Padics.MeasurableSpacePadics
+import FLT.Mathlib.RingTheory.Norm.Defs
+
+open Real Complex Padic MeasureTheory Measure NNReal Set TopologicalSpace
+open scoped Pointwise ENNReal Localization
+
+lemma distribHaarChar_complex (z : ℂˣ) : distribHaarChar ℂ z = ‖(z : ℂ)‖₊ ^ (-2 : ℤ) := by
```

Comment:
Oh crap, really?? The definition of `distribHaarChar` is out by a ^{-1} then :-(